src/fs/metadata_log.cc

```diff
@@ -222,6 +295,24 @@ void metadata_log::memory_only_delete_dir_entry(inode_info::directory& dir, std:
 
     --entry_it->second.directories_containing_file;
     dir.entries.erase(it);
+
+    _ondisk_log_size += log_increase;
+    if (entry_it->second.directories_containing_file == 0) {
+        _rewrite_log_size -= log_adjust;
+    } else {
+        _rewrite_log_size -= log_decrease;
+    }
+}
+
+void metadata_log::memory_only_delete_inode_after_dir_entry(inode_t entry_inode) {
+    size_t log_adjust = ondisk_entry_size<ondisk_delete_inode_and_dir_entry_header>(0) -
+        ondisk_entry_size<ondisk_delete_inode>() - ondisk_entry_size<ondisk_delete_dir_entry_header>(0);
+    memory_only_delete_inode(entry_inode);
+    _ondisk_log_size += log_adjust;
+}
```

Comment:
What is that?
after `after` and before `_dir_entry` should be a verb (adding?, deleting?).